.github/workflows/main.yml

```diff
@@ -32,3 +32,13 @@ jobs:
         SPACE_REGION: ${{ secrets.Spaces_Region}}
         SPACE_ACCESS_KEY_ID: ${{ secrets.Spaces_Key }}
         SPACE_SECRET_ACCESS_KEY: ${{ secrets.Spaces_Secret }}
+    - name: Wait / Sleep
+      uses: kibertoad/wait-action@1.0.1
+      with:
+        time: '2m'
```

Comment:
Why the wait 2 minutes?

Reply:
There is a minimum 1m TTL through DigitalOcean that this accounts for.

Reply:
Adding that to a comment would be nice, so people looking at the code wouldn't be confused